mixpanel_oauth_to_redshift.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"crypto/md5"
+	"database/sql"
+	"encoding/hex"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"sort"
+	"time"
+
+	"github.com/Clever/pathio"
+	_ "github.com/lib/pq" // Postgres Driver
```

Comment:
I would argue that is needed. A package is being imported and not being used so there is no way of knowing what it does. Plus the import path does not provide any information.